src/pages/proposal/active/NewMemberVoting.tsx

```
@@ -50,22 +48,37 @@ const NewMemberVoting = ({
+            component="caption"
+            color="text.secondary"
+            sx={{
+              display: '-webkit-box',
+              WebkitLineClamp: 2,
+              textOverflow: 'ellipsis',
+              overflow: 'hidden',
+              WebkitBoxOrient: 'vertical',
+              textAlign: 'start',
+            }}
+            variant="caption"
+            dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)